tests/system_tests_http1_base.py

```diff
@@ -0,0 +1,1187 @@
+#
```

Comment:
Hmmm... I think this file shouldn't be a "system_tests_*" file since it does not define any TestCases.  It defines a set of HTTP/1.x tests that are shared between the http1_adaptor TestCases and http1_over_tcp TestCases.

Name this fine something like http1_tests.py instead otherwise I think people will be tempted to add HTTP/1.x TestCases here instead of in the proper system_tests_*.py files.